karma.conf.js

```diff
@@ -94,6 +109,8 @@ function combinedLoaders() {
     case 'istanbulInstrumenter':
     case 'tslint':
       return aggregate;
+    case 'ts':
```

Comment:
You can just stack this with the previous cases since it is the same return statement